File: 260427/Date to Date/date-to-date.java
import java.util.Scanner;

public class Main {
    public static int getDays(int m, int d) {
        int[] days = new int[]{0, 31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31};
        int res = d;

        for (int i = 1; i < m; i += 1) {
            res += days[i];
        }

        return res;
    }
    public static void main(String[] args) {
        // Please write your code here.
        Scanner sc = new Scanner(System.in);

        int a = sc.nextInt(),
            b = sc.nextInt(),
            c = sc.nextInt(),
            d = sc.nextInt();

        System.out.println(getDays(c, d) - getDays(a, b) + 1);
    }
}

File: trail2/Date to Date/date-to-date.py
m1, d1, m2, d2 = map(int, input().split())

# Please write your code here.
days = [31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]

day1 = d1 + sum(days[i] for i in range(m1 - 1))
day2 = d2 + sum(days[i] for i in range(m2 - 1))

print(abs(day2 - day1) + 1 if day1 != day2 else 1)